### DISCUSSION TITLE
posts/hack-the-boo-2023/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#40] [General]
# posts/hack-the-boo-2023/

Hack The Boo 2023 Last week, I played Hack The Boo with my team @phis1Ng__. I solved all crypto challenges include 3 in Hack The Boo - Practice, and 2 in Hack The Boo - Competition. All challenges are not too hard but not too easy, so I decided to write a writeup for all of them.
Hack The Boo - Practice 1. Hexoding64 chall.py
from secret import FLAG HEX_CHARS = '0123456789abcdef' B64_CHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/' def to_hex(data): data = int.

https://giapppp.github.io/posts/hack-the-boo-2023/

**Comment by robbert1978** [comment]
That .... du dan
